sql_generators/glean_usage/templates/events_stream_v1.query.sql

```diff
@@ -0,0 +1,64 @@
+{{ header }}
```

Comment:
This looks like it's essentially doing the same as events_unnested but writing it to a table: https://github.com/mozilla/bigquery-etl/blob/main/sql_generators/glean_usage/templates/cross_channel_events_unnested.view.sql